charts/addons/values.yaml

```diff
@@ -52,6 +52,13 @@ aws-load-balancer-controller:
   region: "YOUR_EKS_CLUSTER_REGION"
   ## VPC ID of k8s cluster, required if ec2metadata is unavailable from controller pod
   vpcId: "YOUR_EKS_CLUSTER_VPC_ID"
+  
+  ## Deployments on AWS Gov Cloud requires the image repository to be passed explicitly. Please enable this block for aws gov cloud deployments only.
```

Comment:
The load balancer documentation is provided for reference purpose only. The IAM policy and role creation is common for both conventional commercial cloud deployment and Gov Cloud deployment. If recommended, we can mention this explicitly in Readme.

Reply:
I see...my bad, since missed that this was all pointing to the addons chart changes. That being said, you could make two improvements:

1) in addons values.yaml, move the load balancer link to line 48 so we see it applies to any EKS deployment. I was confused by having the load balancer link in the new GovCloud section ;-)

2) For clarity, you could improve load balancer example in the addons readme file by providing one example for commercial EKS deployments and one for GovCloud, since it's difficult to notate the example to remind us that 'image.repository' not required for  commercial but required for GovCloud deployments.
Commercial EKS Example:

```yaml
aws-load-balancer-controller:
  enabled: true
  clusterName: "YOUR_EKS_CLUSTER_NAME"
  region: "YOUR_EKS_CLUSTER_REGION"
  vpcId: "YOUR_EKS_CLUSTER_VPC_ID"
  serviceAccount:
    annotations:
      eks.amazonaws.com/role-arn: "YOUR_IAM_ROLE_ARN"
```
And then a second one for GovCloud only>>
GovCloud example:

```yaml
aws-load-balancer-controller:
  enabled: true
  clusterName: "YOUR_EKS_CLUSTER_NAME"
  region: "YOUR_EKS_CLUSTER_REGION"
  vpcId: "YOUR_EKS_CLUSTER_VPC_ID"
  image:
    repository: "Amazon EKS Amazon ECR image repository"
  serviceAccount:
    annotations:
      eks.amazonaws.com/role-arn: "YOUR_IAM_ROLE_ARN"
```

Reply:
Updated